ROADMAP.md

```diff
@@ -1,40 +1,17 @@
 # TBD Development Roadmap
 
-Welcome to the TBD Development Roadmap. Here we describe the high-level focus of
-our efforts, chunked into time-boxed deliverables as Milestones. This approach 
-lets teams operate with autonomy, while ensuring that everyone moves in the 
-same direction.
+The roadmap is where we are and where we're going. It's a list of milestones: 
+waypoints that give teams enough direction to make decisions for their projects
+while making for regular sync points. That way, code, programs, and people can
+come together.
 
-Milestones are encoded as a set of goals, defined by the 
-[SMART](https://en.wikipedia.org/wiki/SMART_criteria) model. Goals are:
 
-* Specific
-* Measurable
-* Achievable
-* Relevant
-* Time-bound
-
-Specificity is fluid. Because we are time-boxed, we make delivery estimates on 
-a best-case scenario and adapt as the milestone progresses. Work may have delays
-due to any number of factors, and we employ a blameless culture when goals 
-must be rescoped. When we find our estimates are off, we simply communicate 
-our position to the team via Discussions where we can collectively assess impact
-and adapt as necessary.
-
-Relevance may also be fluid, leaving room to adapt our work to the changing 
-needs of the community. We reduce disruption here by releasing Milestones at 
-frequent, predicable intervals (typically 6-8 weeks).
-
-Milestones will have minimum requirements, and in these cases we prioritize
-work around themes. Missing a theme indicates a loss of focus and can hold up
-others, so we take care to communicate our objectives here and stay true to them
-during development.
-
-Let's go.
 
 ## Milestones
 
-| Milestone                               | Codename            | Ends          |
-|-----------------------------------------|---------------------|---------------|
-| [FUTURE](./milestones/MILESTONE_TBD.md) | Onwards             | None          |
-| [001](./milestones/MILESTONE_001.md)    | Bienvenidos a Miami | 05 April 2022 |
\ No newline at end of file
+| Milestone                                | Codename              | Ends          | Status  | Focus                                    |
+|------------------------------------------|-----------------------|---------------|---------|------------------------------------------|
+| [FUTURE](./milestones/MILESTONE_TBD.md)  | Onwards               | None          | 🟨      |                                          |
+| [003](./milestones/MILESTONE_003.md)     | `TBD`                 | Q1 2023       | 🟨      | MVP of Web5, components feature-complete |
```

Comment:
Confusing naming: Milestone 3 is codenamed `TBD`, but we also have "MILESTONE_TBD" that is different.

Reply:
Yep anything without a number or a name yet I've slated as "TBD" - got a suggestion to make it more clear?

Reply:
I thought milestone 3 codename is literally "TBD".
Consider either:

1. Change codename `Onwards` to also TBD or
1. Change codename `TBD` to something else or
1. Rename MILESTONE_TBD to MILESTONE_FUTURE.md